parsel.js

```diff
@@ -12,8 +12,14 @@ export const TOKENS = {
 const TOKENS_WITH_PARENS = new Set(["pseudo-class", "pseudo-element"]);
 const TOKENS_WITH_STRINGS = new Set([...TOKENS_WITH_PARENS, "attribute"]);
 export const TRIM_TOKENS = new Set(["combinator", "comma"]);
-export const RECURSIVE_PSEUDO_CLASSES = new Set(["not", "is", "where", "has", "matches", "-moz-any", "-webkit-any"]);
+export const RECURSIVE_PSEUDO_CLASSES = new Set(["not", "is", "where", "has", "matches", "-moz-any", "-webkit-any", "nth-child", "nth-last-child"]);
 
+const NTH_CHILD_ARG = /(?<index>[\dn+-]+)\s+of\s+(?<subtree>.+)/
+
+export const RECURSIVE_PSEUDO_CLASSES_ARGS = {
+	"nth-child": NTH_CHILD_ARG,
+	"nth-last-child": NTH_CHILD_ARG
+}
```

Comment:
Nit: I think it would be better to do `RECURSIVE_PSEUDO_CLASSES_ARGS['nth-last-child'] = RECURSIVE_PSEUDO_CLASSES_ARGS['nth-child']` after than to define another constant which looks like it's significant, but is essentially just a helper.